src/scss/components/_button.scss

```
@@ -112,7 +130,9 @@ $button-padding-h: $grid-margin-2;
     $color-border-hover: var(--of-button-primary-hover-color-border),
     $color-active: var(--of-button-primary-active-fg),
     $color-background-active: var(--of-button-primary-active-bg),
-    $color-border-active: var(--of-button-primary-active-color-border)
+    $color-border-active: var(--of-button-primary-active-color-border),
+    $color-border-focus: var(--of-button-primary-focus-color-border),
```

```suggestion
    $color-border-focus: var(--of-button-primary-focus-color-border, var(--utrecht-button-primary-action-focus-border-color)),
```
This one should also fall back to the Utrecht styles. Note that since yesterday the design tokens documentation has been updated on https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css-button-design-tokens--page